.circleci/config.yml

```diff
@@ -111,13 +111,13 @@ jobs:
           name: Deploy to App Engine
           working_directory: ~/workbench/api
           command: |
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+            if [[ "${CIRCLE_BRANCH}" == "master" ]]; then
               ./project.rb deploy \
                 --project all-of-us-workbench-test \
                 --account circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com \
                 --version circle-ci-test \
                 --promote
-            elif [ "${CIRCLE_TAG}" == "v*" ]; then
+            elif [[ "${CIRCLE_TAG}" = v* ]]; then
```

Comment:
v* isn't in quotes?

Reply:
The quotes break it unfortunately.. You can also do `"v"*` I think, but AFAICT it's equivalent.